source/contributor-ladder.md

```diff
@@ -0,0 +1,122 @@
+
+* [User](#user)
+* [Contributor](#contributor)
+* [Committer](#committer)
+* [PMC Member](#pmc-member)
+* [ASF Member](#asf-member)
+* [Officers of the Foundation](#officers-of-the-foundation)
+* [Board of Directors](#board-of-directors)
+
+## User
+
+Everything that we do in an ASF project is for the user. As a user of an
+Apache project, your responsibilities are just to enjoy the software.
+
+We encourage you to join the user mailing list to ask questions, and
+make suggestions. Members of the project are there to help out when you
+run into problems. You can find this list (or lists) by looking under the
+project name on [lists.apache.org](https://lists.apache.org).
+
+As a user, you're expected to abide by the project's [code of
+conduct](https://www.apache.org/foundation/policies/conduct), so that
+interactions are polite, friendly, and helpful.
+
+As you become a power user of the project, you may consider becoming a
+contributor.
+
+## Contributor
+
+While most people think of open source contributors as being coders,
+there are a wider variety of ways that you can contribute to a project.
+
+Most users' first contribution will be to open a ticket on the project's
+issue tracker, or to answer a less experienced user's questions. These
+actions help shape the future direction of the project, and are
+important contributions.
+
+We encourage you to join the project's developer mailing list at this
+time, to engage with the dicussions about the direction that the project
+is taking. You can find this list (or lists) by looking under the
+project name on [lists.apache.org](https://lists.apache.org).
+
+Other contributions may include design, promotion, documentation
+improvements, events, and may other things.
+
+[See more about being a contributor](/contributors).
+
+As you become more involved with contributint to the project, you may
+want to work towards [becoming a
+committer](/contribotirs/becomingacommitter).
+
+## Committer
+
+After making a number of contributions to a project, you may be invited
+to become a committer. Note that the criteria for inviting a contributor
+to become a committer will vary greatly from one project to another.
+
+Once you have been invited to becomg a committer, and have accepted this
+invitation, you are authorized to commit changed directly to the
+project's source code repository. Note, however, that different projects
+may have different social norms around committing changes.
+
+Broadly referred to as CTR (Commit Then Review) and RTC (Review The
+Commit), the exact mechanisms may vary. For example, you may be
+authorized to commit documentation changes without approval, while code
+changes may be submitted as a pull request, for other committers to
+review and approve before they are merged.
+
+[See more information about being a committer](/committers).
+
+## PMC Member
+
+After being a committer for some time, you may be invited to be a member
```

Comment:
again, passive voice obscures who is acting. Would be clearer as "After you have been a committer for some time, the project's PMC may invite you to become a member..." (and the next line would need related changes)